### ISSUE TITLE
Inconsistent yubikey NEO detection

### ISSUE THREAD

**Issue by bendem** [Yubico/yubioath-desktop#317]
- Yubico Authenticator version: HEAD (3accc825a0763d5476704d57b1e0ed72529c1203)
- Operating system and version: macos mojave (was affected in Sierra already)
- YubiKey model and version: NEO, firmware 3.5.0
- Bug description summary: 

When I start the application, I either get `No credentials found`, a password prompt, or the actual tokens available on the key:
<img width="412" alt="screenshot 2018-11-19 at 14 30 45" src="https://user-images.githubusercontent.com/2681677/48710233-fab6a480-ec07-11e8-8c64-ef84ae903a0b.png">
<img width="304" alt="screenshot 2018-11-19 at 14 31 00" src="https://user-images.githubusercontent.com/2681677/48710234-fb4f3b00-ec07-11e8-8b20-da299985d3d9.png">

Steps to reproduce
---
I just start the program and plug the key in, it never works on the first time, but if I unplug a few times, it generally starts working (I think it works faster if I wait a minute before the tries). In the meantime, I've been using my phone and NFC when I need to do something quick, but it's really annoying.

Expected result
---
The program actually shows me my credentials (or an error message, that's fine too)

Other info
---

I ran the application off master with logging, here is the output:
```
(yubioath-desktop) ➜  yubioath-desktop git:(master) ./yubioath-desktop.app/Contents/MacOS/yubioath-desktop --log-level DEBUG
Got library name:  "/usr/local/Cellar/qt/5.11.2/qml/io/thp/pyotherside/libpyothersideplugin.dylib"
2018-11-19T14:45:15+0100 INFO [ykman.logging_setup.setup:59] Initialized logging for ykman version: 1.0.1
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.Descriptor.open_device:75] transports: 0x4, self.mode.transports: 0x7
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:80] Opening driver for serial: None, type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:82] Attempt 1 of 10
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:86] Found driver: <ykman.driver_ccid.CCIDDriver object at 0x11a3412b0>, key_type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:198] Read config from device...
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:213] Failed to read config from device
Traceback (most recent call last):
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/device.py", line 199, in __init__
    config = DeviceConfig(driver.read_config())
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 173, in read_config
    raise NotSupportedError()
ykman.driver.NotSupportedError
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:119] Config data empty/missing
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'0001100000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'00636f629000'
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:227] Identified NEO
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:229] CCID available, probe capabilities...
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OTP , capability: OTP
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040008a0000006472f0001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'5532465f56329000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.U2F , capability: FIDO U2F
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005271002'
2018-11-19T14:45:15+0100 DEBUG [yubikey.refresh:131] Failed to refresh YubiKey
Traceback (most recent call last):
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/device.py", line 199, in __init__
    config = DeviceConfig(driver.read_config())
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 173, in read_config
    raise NotSupportedError()
ykman.driver.NotSupportedError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 201, in send_apdu
    resp, sw1, sw2 = self._conn.transmit(header + body)
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/smartcard/CardConnectionDecorator.py", line 82, in transmit
    return self.component.transmit(bytes, protocol)
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/smartcard/CardConnection.py", line 146, in transmit
    data, sw1, sw2 = self.doTransmit(bytes, protocol)
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/smartcard/pcsc/PCSCCardConnection.py", line 205, in doTransmit
    SCardGetErrorMessage(hresult))
smartcard.Exceptions.CardConnectionException: Failed to transmit with protocol T1. Card is unpowered.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "qrc:///py/yubikey.py", line 124, in refresh
    else TRANSPORT.CCID)
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/descriptor.py", line 87, in open_device
    dev = YubiKey(self, drv)
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/device.py", line 230, in __init__
    usb_supported = driver.probe_capabilities()
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 187, in probe_capabilities
    self.select(aid)
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 215, in select
    return self.send_apdu(0, GP_INS_SELECT, 0x04, 0, aid)
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 204, in send_apdu
    raise CCIDError(e)
ykman.driver_ccid.CCIDError: Failed to transmit with protocol T1. Card is unpowered.
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.__del__:245] Destroy <ykman.driver_ccid.CCIDDriver object at 0x11a3412b0>
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.close:241] Close <ykman.driver_ccid.CCIDDriver object at 0x11a3412b0>
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.Descriptor.open_device:75] transports: 0x4, self.mode.transports: 0x7
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:80] Opening driver for serial: None, type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:82] Attempt 1 of 10
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:86] Found driver: <ykman.driver_ccid.CCIDDriver object at 0x11a3410f0>, key_type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:198] Read config from device...
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:213] Failed to read config from device
Traceback (most recent call last):
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/device.py", line 199, in __init__
    config = DeviceConfig(driver.read_config())
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 173, in read_config
    raise NotSupportedError()
ykman.driver.NotSupportedError
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:119] Config data empty/missing
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'0001100000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'00636f629000'
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:227] Identified NEO
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:229] CCID available, probe capabilities...
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OTP , capability: OTP
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040008a0000006472f0001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'5532465f56329000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.U2F , capability: FIDO U2F
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005271002'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'6d00'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:193] Missing applet: aid: AID.U2F_YUBICO , capability: FIDO U2F
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040005a000000308'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'61114f0600001000010079074f05a0000003089000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.PIV , capability: PIV
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040006d27600012401'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'9000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OPGP , capability: OpenPGP
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272101'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'790301000171087cb4565782daaaa39000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OATH , capability: OATH
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272101'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'790301000171087cb4565782daaaa39000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.__del__:245] Destroy <ykman.driver_ccid.CCIDDriver object at 0x11a3410f0>
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.close:241] Close <ykman.driver_ccid.CCIDDriver object at 0x11a3410f0>
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.Descriptor.open_device:75] transports: 0x4, self.mode.transports: 0x7
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:80] Opening driver for serial: None, type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:82] Attempt 1 of 10
2018-11-19T14:45:15+0100 DEBUG [ykman.descriptor.open_device:86] Found driver: <ykman.driver_ccid.CCIDDriver object at 0x11a3412e8>, key_type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:198] Read config from device...
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:213] Failed to read config from device
Traceback (most recent call last):
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/device.py", line 199, in __init__
    config = DeviceConfig(driver.read_config())
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 173, in read_config
    raise NotSupportedError()
ykman.driver.NotSupportedError
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:119] Config data empty/missing
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'0001100000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'00636f629000'
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:227] Identified NEO
2018-11-19T14:45:15+0100 DEBUG [ykman.device.__init__:229] CCID available, probe capabilities...
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OTP , capability: OTP
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040008a0000006472f0001'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'5532465f56329000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.U2F , capability: FIDO U2F
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005271002'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'6d00'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:193] Missing applet: aid: AID.U2F_YUBICO , capability: FIDO U2F
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040005a000000308'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'61114f0600001000010079074f05a0000003089000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.PIV , capability: PIV
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040006d27600012401'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'9000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OPGP , capability: OpenPGP
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272101'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'790301000171087cb4565782daaaa39000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OATH , capability: OATH
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272101'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'790301000171087cb4565782daaaa39000'
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.__del__:245] Destroy <ykman.driver_ccid.CCIDDriver object at 0x11a3412e8>
2018-11-19T14:45:15+0100 DEBUG [ykman.driver_ccid.close:241] Close <ykman.driver_ccid.CCIDDriver object at 0x11a3412e8>
2018-11-19T14:45:16+0100 DEBUG [ykman.descriptor.Descriptor.open_device:75] transports: 0x4, self.mode.transports: 0x7
2018-11-19T14:45:16+0100 DEBUG [ykman.descriptor.open_device:80] Opening driver for serial: None, type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:16+0100 DEBUG [ykman.descriptor.open_device:82] Attempt 1 of 10
2018-11-19T14:45:16+0100 DEBUG [ykman.descriptor.open_device:86] Found driver: <ykman.driver_ccid.CCIDDriver object at 0x11a341518>, key_type: YUBIKEY.NEO, mode: OTP+FIDO+CCID
2018-11-19T14:45:16+0100 DEBUG [ykman.device.__init__:198] Read config from device...
2018-11-19T14:45:16+0100 DEBUG [ykman.device.__init__:213] Failed to read config from device
Traceback (most recent call last):
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/device.py", line 199, in __init__
    config = DeviceConfig(driver.read_config())
  File "/Users/benjamin/projects/github/yubioath-desktop/yubioath-desktop.app/Contents/MacOS/pymodules/ykman/driver_ccid.py", line 173, in read_config
    raise NotSupportedError()
ykman.driver.NotSupportedError
2018-11-19T14:45:16+0100 DEBUG [ykman.device.__init__:119] Config data empty/missing
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'0001100000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'00636f629000'
2018-11-19T14:45:16+0100 DEBUG [ykman.device.__init__:227] Identified NEO
2018-11-19T14:45:16+0100 DEBUG [ykman.device.__init__:229] CCID available, probe capabilities...
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272001'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'030500018507060f00009000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OTP , capability: OTP
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040008a0000006472f0001'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'5532465f56329000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.U2F , capability: FIDO U2F
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005271002'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'6d00'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.probe_capabilities:193] Missing applet: aid: AID.U2F_YUBICO , capability: FIDO U2F
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040005a000000308'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'61114f0600001000010079074f05a0000003089000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.PIV , capability: PIV
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040006d27600012401'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'9000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OPGP , capability: OpenPGP
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272101'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'790301000171087cb4565782daaaa39000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.probe_capabilities:190] Found applet: aid: AID.OATH , capability: OATH
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a4040007a0000005272101'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'790301000171087cb4565782daaaa39000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a400010a74080000000003109ff2'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:202] RECV: b'710d4769744875623a62656e64656d760506655f8d0d7119476f6f676c653a62656e64656d626440676d61696c2e636f6d76050663abba76711b4369747573436c6f75643a6f6e6c696e654062656e64656d2e62657605060a680eb37123476f6f676c653a62656e6a616d696e2e64656d6172746561754079656c69646f2e6265760506097da1f671194769744875623a62656e6a616d696e2d64656d6172746561757605062dd13331712c416d617a6f6e205765622053657276696365733a62656e6a616d696e2e64656d6172746561754061326c696f760506293d0314710f4353414d2d4c6f67696e3a6e756c6c7605063c38f8cc71254461736861fd'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.send_apdu:200] SEND: b'00a5000000'
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.__del__:245] Destroy <ykman.driver_ccid.CCIDDriver object at 0x11a341518>
2018-11-19T14:45:16+0100 DEBUG [ykman.driver_ccid.close:241] Close <ykman.driver_ccid.CCIDDriver object at 0x11a341518>
```

```
(yubioath-desktop) ➜  yubioath-desktop git:(master) pip freeze
asn1crypto==0.24.0
cffi==1.11.5
Click==7.0
cryptography==2.4.1
fido2==0.4.0
idna==2.7
pycparser==2.19
pyOpenSSL==18.0.0
pyscard==1.9.7
pyusb==1.0.2
qr==0.1
six==1.11.0
yubikey-manager==1.0.1
```

**Comment by dagheyman** [comment]
Hi, thanks for the report. Do you get the same result with 4.3.5 (released today) ?

**Comment by bendem** [comment]
Can you provide a `.pkg` file so I can try it out?

**Comment by dagheyman** [comment]
https://developers.yubico.com/yubioath-desktop/Releases/

**Comment by bendem** [comment]
If that helps, here is what happens (I plug and unplug the key a few times):
![screen recording 2019-01-03 at 12 26 33](https://user-images.githubusercontent.com/2681677/50635710-8768b800-0f53-11e9-8e64-4b2e4e03453d.gif)

**Comment by dagheyman** [comment]
Thanks for confirming and for the extra details. That the password prompt is shown is definitely a bug, and in addition to that the error handling when a CCID connections fails needs to be improved.

**Comment by davewilliamson** [comment]
I'm getting exactly the same issue!

**Comment by dagheyman** [comment]
Could be related to this issue: https://github.com/Yubico/yubikey-manager/issues/137
Need to look into error handling in the CCID + NEO + macOS scenario.

**Comment by evilpete** [comment]
I am also seeing errors on MacOS also with updates stoping

So today I upgraded from yubioath-desktop-4.3.4-mac to yubioath-desktop-4.3.5-mac and now it claims the the yubikey is not in CCID mode,  I have multiple keys and they I checked that they both are)

**Comment by jhrmnn** [comment]
It seems I have run into a similar issue (macOS 10.14.4). Another symptom is that the Authenticator does not shut down normally, it hangs. I have to kill the process.

**Comment by bendem** [comment]
I'd like to report that seems resolved in version `10.14.6 (18G95)`. I have not encountered it in days.
